packages/env/lib/cli.js

```diff
@@ -124,6 +124,12 @@ module.exports = function cli() {
 		},
 		withSpinner( env.run )
 	);
+	yargs.command(
+		'destroy',
+		wpRed( 'destroy the WordPress environment.' ),
```

Comment:
Could you expand this a little more to say that it removes the downloaded files and docker containers?